langchain/chains/llm_math/base.py

```diff
@@ -40,6 +44,19 @@ class Config:
         extra = Extra.forbid
         arbitrary_types_allowed = True
 
+    @root_validator(pre=True)
+    def raise_deprecation(cls, values: Dict) -> Dict:
```

Comment:
@agola11 @hwchase17 is this good way to handle refactoring to take in llm_chain instead of llm?

Reply:
yep looks fine